rest/handler.go

```diff
@@ -294,11 +303,11 @@ func (h *handler) validateAndWriteHeaders(method handlerMethod, accessPermission
 		return base.HTTPErrorf(http.StatusUnsupportedMediaType, "Unsupported Content-Encoding; use gzip")
 	}
 
-	if h.shouldShowProductVersion() {
-		h.setHeader("Server", base.VersionString)
-	} else {
-		h.setHeader("Server", base.ProductNameString)
-	}
+	//if h.shouldShowProductVersion() {
```

Comment:
I'm not sure what's going on with this change - is this intentional?

Reply:
mistake